particles/Tutorial/Kotlin/Blackjack/Dealer.kt

```diff
@@ -0,0 +1,38 @@
+package arcs.tutorials.blackjack
+
+import arcs.sdk.Handle
+import arcs.sdk.Utils.log
+
+class Dealer : AbstractDealer() {
+    val name = "Dealer"
+
+    init {
+        eventHandler("onHit") {
+            cardRequest.set(Dealer_CardRequest(player = name))
+            log("Hit.")
+        }
+        eventHandler("onStand") {
+            log("Stand to be implemented.")
+        }
+    }
+    override fun getTemplate(slotName: String) = """
+        ${name}: <button type="button" on-click="onHit"> Hit </button> 
+        <button type="button" on-click="onStand"> Stand </button>
+        <span>{{hand}}</span> 
+     """.trimIndent()
```

Comment:
Indentation is a bit funky here.  Should be 4 more spaces than the `override fun` indentation level for the quotes, and another four spaces for the body within the quotes.